src/activities/quizzes/attempts/QuizAttemptsEntity.js

```diff
@@ -106,6 +128,33 @@ export class QuizAttemptsEntity extends Entity {
 		return this._entity && this._entity.getSubEntityByClass(Classes.quizzes.attempts.retakeIncorrectOnly);
 	}
 
+	/**
+	 * @returns {object} quiz attempt conditions sub-entity
+	 */
+	getAttemptConditionsSubEntity() {
+		if (this._entity.hasSubEntityByClass(Classes.quizzes.attempts.attemptConditions) && this._entity.hasSubEntityByClass('collection')) {
+			return this._entity && this._entity.getSubEntityByClass(Classes.quizzes.attempts.attemptConditions);
+		}
+
+		return null;

```

Comment:
Do we need to return `null`? Can the function just return nothing, i.e. undefined?